ci/it/testcases/test_only_common_table.go

```diff
@@ -0,0 +1,67 @@
+// Copyright Quesma, licensed under the Elastic License 2.0.
+// SPDX-License-Identifier: Elastic-2.0
+
+package testcases
+
+import (
+	"context"
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"testing"
+)
+
+type OnlyCommonTableTestcase struct {
+	IntegrationTestcaseBase
+}
+
+func NewOnlyCommonTableTestcase() *OnlyCommonTableTestcase {
+	return &OnlyCommonTableTestcase{
+		IntegrationTestcaseBase: IntegrationTestcaseBase{
+			ConfigTemplate: "quesma-only-common-table.yml.template",
+		},
+	}
+}
+
+func (a *OnlyCommonTableTestcase) SetupContainers(ctx context.Context) error {
+	containers, err := setupAllContainersWithCh(ctx, a.ConfigTemplate)
+	a.Containers = containers
+	return err
+}
+
+func (a *OnlyCommonTableTestcase) RunTests(ctx context.Context, t *testing.T) error {
+
+	t.Run("test alter virtual table", func(t *testing.T) { a.testAlterVirtualTable(ctx, t) })
+	return nil
+}
+
+func (a *OnlyCommonTableTestcase) testAlterVirtualTable(ctx context.Context, t *testing.T) {
+
+	reloadTables := func() {
+		resp, body := a.RequestToQuesma(ctx, t, "POST", "/_quesma/reload-tables", nil)
```

Comment:
We should put that method under our admin port, not on ElasticSearch frontend.

As we are rushing with release fine to do in next PR.